src/SprykerEco/Zed/Optivo/Dependency/Facade/OptivoToSalesFacadeInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+namespace SprykerEco\Zed\Optivo\Dependency\Facade;
+
+use Generated\Shared\Transfer\OrderTransfer;
+
+interface OptivoToSalesFacadeInterface
+{
+    /**
+     * @param int $idOrder
+     *
+     * @return \Generated\Shared\Transfer\OrderTransfer
+     */
+    public function getOrderByIdSalesOrder(int $idOrder): OrderTransfer;
```

Comment:
Please use exactly the same method typehinting like in SalesFacadeInterface.

Reply:
cannot understand you, please, explain it